# Statistical Learning_R2_Project2_Titan Insurance case study

The Titan Insurance Company has just installed a new incentive payment scheme for its lift policy sales force. It wants to have an early view of the success or failure of the new scheme. Indications are that the sales force is selling more policies, but sales always vary in an unpredictable pattern from month to month and it is not clear that the scheme has made a significant difference.

Life Insurance companies typically measure the monthly output of a salesperson as the total sum assured for the policies sold by that person during the month. For example, suppose salesperson X has, in the month, sold seven policies for which the sums assured are £1000, £2500, £3000, £5000, £10000, £35000. X's output for the month is the total of these sums assured, £61,500. Titan's new scheme is that the sales force receives low regular salaries but are paid large bonuses related to their output (i.e. to the total sum assured of policies sold by them). The scheme is expensive for the company, but they are looking for sales increases which more than compensate. The agreement with the sales force is that if the scheme does not at least break even for the company, it will be abandoned after six months.

The scheme has now been in operation for four months. It has settled down after fluctuations in the first two months due to the changeover.

To test the effectiveness of the scheme, Titan have taken a random sample of 30 salespeople measured their output in the penultimate month prior to changeover and then measured it in the fourth month after the changeover (they have deliberately chosen months not too close to the changeover). The outputs of the salespeople are shown in Table 1


SALESPERSON	Old Scheme (in thousands)	New Scheme (in thousands)
1	57	62
2	103	122
3	59	54
4	75	82
5	84	84
6	73	86
7	35	32
8	110	104
9	44	38
10	82	107
11	67	84
12	64	85
13	78	99
14	53	39
15	41	34
16	39	58
17	80	73
18	87	53
19	73	66
20	65	78
21	28	41
22	62	71
23	49	38
24	84	95
25	63	81
26	77	58
27	67	75
28	101	94
29	91	100
30	50	68




Questions
1. Find the mean of old scheme and new scheme column. (5 points)
2. Use the five percent significance test over the data to determine the p value to check new scheme has significantly raised outputs? (8 points)
3. What conclusion does the test (p-value) lead to? (2 points)
4. Suppose it has been calculated that in order for Titan to break even, the average output must increase by £5000 in the scheme compared to the old scheme. If this figure is alternative hypothesis, what is:
        a) The probability of a type 1 error? (2 points)

        b) What is the p- value of the hypothesis test if we test for a difference of $5000? (8 points)

        c) Power of the test (5 points)

In [32]:
# Load data from csv
import pandas as pd
file_path = "Titan Insurance Company.csv"
tic = pd.read_csv(file_path)

In [33]:
# describe tic
tic.describe

<bound method NDFrame.describe of     SALESPERSON  Old Scheme (in thousands)  New Scheme (in thousands)
0             1                         57                         62
1             2                        103                        122
2             3                         59                         54
3             4                         75                         82
4             5                         84                         84
5             6                         73                         86
6             7                         35                         32
7             8                        110                        104
8             9                         44                         38
9            10                         82                        107
10           11                         67                         84
11           12                         64                         85
12           13                         78              

In [34]:
tic.shape

(30, 3)

1. Find the mean of old scheme and new scheme column. (5 points)

In [35]:
tic_old_mean = tic["Old Scheme (in thousands)"].mean()
tic_old_mean

68.03333333333333

In [36]:
tic_new_mean = tic["New Scheme (in thousands)"].mean()
tic_new_mean

72.03333333333333

Answer: Mean of mean of old scheme is 68.03 and new scheme is 72.03

2.Use the five percent significance test over the data to determine the p value to check new scheme has significantly raised outputs? (8 points)

In [37]:
from scipy.stats import ttest_rel
ttest = ttest_rel(tic["New Scheme (in thousands)"],tic["Old Scheme (in thousands)"])
ttest

Ttest_relResult(statistic=1.5559143823544377, pvalue=0.13057553961337662)

Assume alpha = 0.05 with 95% confidence level for accepting Alternate Hypothesis

Null Hypothesis : There is no significant difference between data sets

Alternate Hypothesis : There is a significant difference

P-value is 0.13


3.What conclusion does the test (p-value) lead to? (2 points)

Answer: As p-value is more than alpha so we accept Alternate hypothesis.
There is a significant difference in output

4.Suppose it has been calculated that in order for Titan to break even, the average output must increase by £5000 in the scheme compared to the old scheme. If this figure is alternative hypothesis, what is:

 a) The probability of a type 1 error? (2 points)

Answer

Null Hypothesis : Current avg output is sufficient

Alternate Hypothesis : Current avg output needs to increase by 5000GBP

In [38]:
#Use 1 sample ttest with an increas of mean by 5 in old scheme
import scipy.stats

diff_1s_ttest_pValue = scipy.stats.ttest_1samp(tic["Old Scheme (in thousands)"],tic["Old Scheme (in thousands)"].mean()+5)
diff_1s_ttest_pValue

Ttest_1sampResult(statistic=-1.338783455563436, pvalue=0.1910421709623265)

Answer: Probability of Type1 error is 19.1%

 b) What is the p- value of the hypothesis test if we test for a difference of $5000? (8 points)

In [39]:
# Get the difference between Old and new scheme
tic["scheme_diff"] = tic["New Scheme (in thousands)"] - tic["Old Scheme (in thousands)"]
tic.head()

SALESPERSON  Old Scheme (in thousands)  New Scheme (in thousands)  \
0            1                         57                         62   
1            2                        103                        122   
2            3                         59                         54   
3            4                         75                         82   
4            5                         84                         84   

   scheme_diff  
0            5  
1           19  
2           -5  
3            7  
4            0

In [40]:
#Use 1 sample ttest with mean of 5
import scipy.stats

diff_1s_ttest_pValue = scipy.stats.ttest_1samp(tic.scheme_diff, 5)
diff_1s_ttest_pValue

Ttest_1sampResult(statistic=-0.3889785955886094, pvalue=0.7001334912613286)

Answer: Pvalue is 0.70 for difference of 5K

c) Power of the test (5 points)

# Attempting this question though this has not been taught in the class and Rajesh had mentioned that he has never used Beta so not sure what is the purpose of this question

In [41]:
# Calculate effect_size
tic_effect_size = (tic_new_mean - tic_old_mean)/tic['New Scheme (in thousands)'].std()
tic_effect_size

0.1662344919883238

In [42]:
# Get power of ttest for number of observations nobs = 30 and aplha = 0.05
import statsmodels.stats.power as smp
smp.ttest_power(tic_effect_size, nobs=30, alpha=0.05)

0.14246837920902358